pages/implementation/read-implementation-diff.mdx

```diff
@@ -0,0 +1,81 @@
+---
+layout: 'guide'
+title: 'How to read the implementation diff'
+---
+
+import Link from '../../components/Link';
+
+First thing you need to know is that tracking design happens on branches. When a new feature is built, the analytics manager opens a branch in Avo and plans all the new tracking there.
+This allows us to show you the diff - what should be implemented for a specific feature. 
+
```

Comment:
Should we add a screenshot here of the "Share instructions" panel from the diff view, so people know exactly what we're referring to here?